config/dataset/test_ci_segmentation_dataset.yaml

```diff
@@ -0,0 +1,17 @@
+# @package _global_
+dataset:
+  name:
+  modalities: RGB
+  input_dim: 256
+  overlap:
+  min_annot_perc:
+  raw_data_csv: ${general.raw_data_csv}
+  raw_data_dir: ${general.raw_data_dir}
+  sample_data_dir: ${general.sample_data_dir}
+  class_name: # will follow in the next version
+  classes_dict: {'Building':1}
```

Comment:
Replacing number of classes by this dictionary is a cool change (though not directly related to hydra - could've been in separate PR :) ). I was just talking about with YvesM and Mat. We will build on this and use it during inference and post-processing.